core/rawdb/schema.go

```diff
@@ -145,6 +154,11 @@ func txLookupKey(hash common.Hash) []byte {
 	return append(txLookupPrefix, hash.Bytes()...)
 }
 
+// stakingTxLookupKey = stakingTxLookupPrefix + hash
+func stakingTxLookupKey(hash common.Hash) []byte {
```

Comment:
why `TxLookupKey` is not sufficient? Do you expect normal tx hash and staking tx hash to collide? If so, different prefix is required, otherwise not. Also, note that going forward when we unify normal and staking transactions, all these additional code specific to staking tx will be deleted anyways.

Reply:
hmm right. with unification, this is one of the speicifc staking tx code to be deleted.
will make the change to uses 'txlookupkey'